sdk/platform.go

```diff
@@ -149,15 +149,17 @@ func (pf *ProjectPlatform) HideSecrets() {
 
 // MergeWith merge two config
 func (config *PlatformConfig) MergeWith(cfg PlatformConfig) {
-	if config == nil {
+	if *config == nil {
```

Comment:
i think you should test `config == nil` i.e the pointer to the underlying map is nil and `*config == nil` i.e. the pointer is valid but the the underlying map is nil (has never been initialized)